sites/fast-website/src/app/components/section-header/section-header.styles.ts

```diff
@@ -9,4 +13,14 @@ export const SectionHeaderStyles = css`
         box-sizing: border-box;
         text-align: center;
     }
-`;
+
+    :host ::slotted(.section-badge) {
+        color: var(--accent-color);
+    }
+`.withBehaviors(
+    cssCustomPropertyBehaviorFactory(
+        "accent-color",
+        d => d.accentBaseColor,
+        DesignSystemProvider.findProvider
```

Comment:
This should be a reference to the `WebsiteDesignSystemProvider` since that is the custom element constructor you're expecting to resolve this recipe.

Reply:
actually - I *think* this should just use the `accentForegroundRestBehavior` from `@microsoft/fast-components`